Packs/CommonScripts/Scripts/DockerHardeningCheck/DockerHardeningCheck.py

```diff
@@ -150,9 +159,50 @@ def check_cpus(num_cpus: int) -> str:
     return ""
 
 
+def get_default_gateway():
+    res = subprocess.check_output(["ip", "route", "list"], text=True, stderr=subprocess.STDOUT)
+    LOG(f'result of ip route list: {res}')
+    line1 = res.splitlines()[0]
+    if not line1.startswith('default via'):
```

Comment:
Maybe protect against case discrepancies?
```suggestion
    if not line1.lower().startswith('default via'):
```